prisma/migrations/20221127044819_added_access_token/migration.sql

```diff
@@ -0,0 +1,112 @@
+-- CreateTable
```

Comment:
We dont need to add the migrations files for now. As the migrations files will differ for everyone and hence doesnt add any value in maintaining here.